apps/edxapp/templates/services/cms/_dc_base.yml.j2

```diff
@@ -91,15 +91,15 @@ spec:
             command:
               - "/bin/bash"
               - "-c"
-              - "python manage.py {{ service_variant }} celery inspect ping -d celery@{{ celery_worker.name }}.%$(hostname)"
+              - "celery -b redis://redis:{{ redis_app_port }} inspect ping -d celery@{{ celery_worker.name }}.%$(hostname)"
           initialDelaySeconds: 120
-          periodSeconds: 400
+          periodSeconds: 900
```

Comment:
Maybe you don't need to increase it anymore now that you optimized it?

Reply:
Ypu, but I think that: i. we should have a period that corresponds to a minute-based granularity (_e.g._ 10 minutes), and, ii. we agreed that given the number of replicates and pods affinity, something like `x>=10` minutes should be ok to restart a worker that handles asynchronous jobs. Deal?

Reply:
ok